### DISCUSSION TITLE
Version 1.21.0

### DISCUSSION THREAD

**Discussion by MM2-0** [MM2-0/Kvaesitso#231] [Announcements]
- Advanced layout options: you can now configure the search bar position, the arrangement of search results and the gesture to open the search independently
- Gestures: added configurable gestures including double tap to sleep[^1]
- Added a hardcoded list of music players (namely, Tidal, Bandcamp, Qobuz and Plexamp) that were formerly not recognized as "music apps" and required the "Restrict to music apps" setting to be disabled in order to work with the music widget. Let me know if you use any other music apps that are not properly recognized, I will add them to the list.
- Bug fixes

[^1] Double tap to sleeps is only available on Android 9+

<hr /><em>This discussion was created from the release <a href='https://github.com/MM2-0/Kvaesitso/releases/tag/v1.21.0'>Version 1.21.0</a>.</em>

**Comment by UcraNerd** [comment]
I want to marry thi-this awesome foss launcher; "cry about it pixel launcher".
